velox/common/base/PeriodicStatsReporter.cpp

```diff
@@ -115,6 +115,10 @@ void PeriodicStatsReporter::reportAllocatorStats() {
   RECORD_METRIC_VALUE(
       kMetricAllocatedMemoryBytes,
       (velox::memory::AllocationTraits::pageBytes(allocator_->numAllocated())));
+  RECORD_METRIC_VALUE(
+      kMetricTotalUsedBytes,
+      (velox::memory::AllocationTraits::pageBytes(
+          allocator_->totalUsedBytes())));
```

Comment:
The API returns bytes directly, if wrapping around velox::memory::AllocationTraits::pageBytes() you'll get bytes multiplied by page size 4096